packages/lexical-utils/src/index.ts

```diff
@@ -524,5 +524,7 @@ export function objectKlassEquals<T>(
   object: unknown,
   objectClass: ObjectKlass<T>,
 ): boolean {
-  return Object.getPrototypeOf(object).constructor.name === objectClass.name;
+  return object
```

Comment:
I presume you're checking null values here. Can we use triple equality?
 
```suggestion
  return typeof object === 'object'
```

Reply:
Using `object !== null` as `typeof null` returned `object`.